### DISCUSSION TITLE
Study tools do not appear for Teaching assistants

### DISCUSSION THREAD

**Discussion by Pinkvinus** [PhilipFlyvholm/learnit-plus-plus#12] [Ideas]
As stated, the section study tools doesn't appear for me. In it's place is teaching tools, which I have little use of. I would guess this is due to my position as a TA, but would much prefer to have study tools rather than teaching tools.

Vanilla LearnIT:
![image](https://github.com/PhilipFlyvholm/learnit-plus-plus/assets/74830754/68e38e82-b540-4b14-9e89-065c7f704fad)

LearnIT++:
![image](https://github.com/PhilipFlyvholm/learnit-plus-plus/assets/74830754/b28f3ab2-4633-41be-9ee6-a3b6f8262ef0)

**Comment by PatrickMatthiesen** [comment]
We will look at this when we have settings up and running

**Comment by PhilipFlyvholm** [comment]
Future reference for study tools:
![image](https://github.com/PhilipFlyvholm/learnit-plus-plus/assets/36300927/fa33a55a-8cec-4a18-ac33-ab62af1c3083)
